scripts/cluster_network.py

```diff
@@ -379,6 +379,8 @@ def consense(x):
                                                extended_link_costs=hvac_overhead_cost,
                                                focus_weights=focus_weights)
 
+    update_p_nom_max(n)
```

Comment:
I am just wondering, is this one still necessary? Because we already ensured that p_nom_max >= p_nom_min right?

Reply:
I was wondering that, too. But the clustering (if potential: conservative) aggregates the p_nom_max values using the minimum value, whilst the p_nom_min ones are now summed up. So there might be cases where this might be necessary.

However, by default, thepotential is 'simple', hence the p_nom_max values are summed.